src/main/steps/respondent/hub-page/content.ts

```diff
@@ -79,9 +79,9 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
   },
   conditionalOrderPronounced: {
     line1: `You have been granted a ‘conditional order’ by the court. Your conditional order was formally pronounced
-    (read out) by a judge at ${userCase.coCourt === ConditionalOrderCourt.BIRMINGHAM ? birmingham : buryStEdmunds} on ${
-      userCase.coDateOfHearing
-    }.
+    (read out) by a judge at ${
+      userCase.coCourt === ConditionalOrderCourt.BIRMINGHAM ? birmingham : buryStEdmunds
```

Comment:
is this needed? wouldn't just using the enum work? or if not toString()

Reply:
nope, as the enums are more like individual variables